src/components/views/dialogs/spotlight/SpotlightDialog.tsx

```diff
@@ -1065,6 +1065,56 @@ const SpotlightDialog: React.FC<IProps> = ({ initialText = "", initialFilter = n
             </>
         );
     }
+    // calculate no of element to jump based on arrow key or page up and page down
+    const getNoOfEleToJump = ({
+        currentIndex,
+        totalElements,
+        noOfJump,
+    }: {
+        currentIndex: number;
+        totalElements: number;
+        noOfJump: number;
+    }) => {
+        if (currentIndex + noOfJump < 0) {
+            return 0;
+        } else if (currentIndex + noOfJump > totalElements - 1) {
+            return totalElements - 1;
+        } else {
+            return currentIndex + noOfJump;
+        }
+    };
+
+    const findNextElementIndex = ({
+        accessibilityAction,
+        idx,
+        refs,
+    }: {
+        accessibilityAction: KeyBindingAction;
+        idx: number;
+        refs: RefObject<HTMLElement>[];
+    }) => {
+        let noOfEleToJump: number = 1;
+        if (accessibilityAction === KeyBindingAction.ArrowUp || accessibilityAction === KeyBindingAction.ArrowDown) {
+            // move to next element in either direct using arrow key
+            noOfEleToJump = getNoOfEleToJump({
+                currentIndex: idx,
+                totalElements: refs.length,
+                noOfJump: accessibilityAction === KeyBindingAction.ArrowUp ? -1 : 1,
+            });
+        } else if (
+            accessibilityAction === KeyBindingAction.PageUp ||
+            accessibilityAction === KeyBindingAction.PageDown
+        ) {
+            // jump 3 next element in either direct using page up and page down key
```

Comment:
1. Why 3? The number of items spotlight shows at once is 12
2. Please make this a constant so it can be changed if necessary

Reply:
Page up/down should jump by an entire page, this is quite important to match accessibility expectations.

Reply:
The number of items will vary based on the user's laptop (as we are not targeting mobile screens) screen sizes. In some small-screen laptops, it will only show 6 people or 2-3 public rooms at a time. It will be tricky to handle.

Reply:
This is still not right, we need to match the behaviour expected for accessibility reasons